install-flox.sh

```diff
@@ -14,7 +14,12 @@ nix profile install --impure \
       --accept-flake-config \
       'github:flox/floxpkgs#flox.fromCatalog'
 
+# Check it runs
+$HOME/.nix-profile/bin/flox --version
+
+# This may not already be included if Nix didn't come from install-nix-action
```

Comment:
can we undo the double negation here :D

```suggestion
# This might already be set e.g. if Nix was installed using install-nix-action.
# If Nix was already installed through other means (e.g. on a hosted runner)
# this path might be missing, so we include it to be safe.
```